File: app/src/main/java/com/stathis/elmepaunivapp/ui/students/model/Schedule.java
package com.stathis.elmepaunivapp.ui.students.model;

public class Schedule extends Object {

    String mainTxt;
    String buttonTxt;

    public Schedule(String mainTxt, String buttonTxt) {
        this.mainTxt = mainTxt;
        this.buttonTxt = buttonTxt;
    }

    public String getMainTxt() {
        return mainTxt;
    }

    public void setMainTxt(String mainTxt) {
        this.mainTxt = mainTxt;
    }

    public String getButtonTxt() {
        return buttonTxt;
    }

    public void setButtonTxt(String buttonTxt) {
        this.buttonTxt = buttonTxt;
    }
}

File: app/src/main/java/com/stathis/elmepaunivapp/ui/students/model/Schedule.kt
package com.stathis.elmepaunivapp.ui.students.model

import android.widget.Button
import com.stathis.elmepaunivapp.model.LocalModel

data class Schedule(

    val mainTxt : String,
    val mainButton : String

) : LocalModel {
    override fun equalsContent(obj: LocalModel): Boolean = false
}
